sdk/netapp/Microsoft.Azure.Management.NetApp/src/Customizations/NetAppResourceOperationsExtensions.cs

```diff
@@ -0,0 +1,331 @@
+        {
+            //using (var _result = await ((NetAppResourceOperations)operations).CheckFilePathAvailabilityWithHttpMessagesAsync(location, name, type, resourceGroup, null, cancellationToken).ConfigureAwait(false))
+            using (var _result = await ((NetAppResourceOperations)operations).CheckFilePathAvailabilityWithHttpMessagesAsync(location, name, type, resourceGroup, null, cancellationToken).ConfigureAwait(false))
+            {
+                return _result.Body;
+            }
+        }
+
+    }
+    internal partial class NetAppResourceOperations : IServiceOperations<AzureNetAppFilesManagementClient>, INetAppResourceOperations
+    {
+
+        /// <summary>
+        /// Check file path availability
+        /// </summary>
+        /// <remarks>
+        /// Check if a file path is available.
+        /// </remarks>
+        /// <param name='location'>
+        /// The location
+        /// </param>
+        /// <param name='name'>
+        /// Resource name to verify.
+        /// </param>
+        /// <param name='type'>
+        /// Resource type used for verification. Possible values include:
+        /// 'Microsoft.NetApp/netAppAccounts',
+        /// 'Microsoft.NetApp/netAppAccounts/capacityPools',
+        /// 'Microsoft.NetApp/netAppAccounts/capacityPools/volumes',
+        /// 'Microsoft.NetApp/netAppAccounts/capacityPools/volumes/snapshots'
+        /// </param>
+        /// <param name='resourceGroup'>
+        /// Resource group name.
+        /// </param>
+        /// <param name='customHeaders'>
+        /// Headers that will be added to request.
+        /// </param>
+        /// <param name='cancellationToken'>
+        /// The cancellation token.
+        /// </param>
+        /// <exception cref="CloudException">
+        /// Thrown when the operation returned an invalid status code
+        /// </exception>
+        /// <exception cref="SerializationException">
+        /// Thrown when unable to deserialize the response
+        /// </exception>
+        /// <exception cref="ValidationException">
+        /// Thrown when a required parameter is null
+        /// </exception>
+        /// <exception cref="System.ArgumentNullException">
+        /// Thrown when a required parameter is null
+        /// </exception>
+        /// <return>
+        /// A response object containing the response body and response headers.
+        /// </return>
+        public async Task<AzureOperationResponse<CheckAvailabilityResponse>> CheckFilePathAvailabilityWithHttpMessagesAsync(string location, string name, string type, string resourceGroup, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            string apiVersion = "2021-02-01";
```

Comment:
So there is no way to translate the old parameters into the new api?

Reply:
Unfortunaly there is no way to work out the subnet id from the old parameters to satisfy the new API